test/e2e/scenarios/no_np.go

```diff
@@ -0,0 +1,18 @@
+// Test the scenario where there are no network policies
+// implemented in which case all pods should be able to
+// communicate with each other
+
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+	// . "github.com/onsi/gomega"
+)
+
+var _ = PDescribe("No network policy is implemented", func() {
+	PIt("Should allow communication between all pods", func() {
+		By("creating pod1 with label1 in cluster1 in namespace1")
```

Comment:
here technically you don't need to care about label or namespace, only cluster,

Reply:
Yes. This is just for uniformity